2004-03-20  Petter Reinholdtsen  <pere@hungry.com>

	* localedata/locales/nb_NO: add Norwegian transliteration.

---
 localedata/locales/nb_NO |   15 +++++++++++++++
 1 file changed, 15 insertions(+)

--- a/localedata/locales/nb_NO
+++ b/localedata/locales/nb_NO
@@ -127,6 +127,21 @@
 
 translit_start
 include  "translit_combining";""
+
+% Norwegian transliteration
+% LATIN CAPITAL LETTER A WITH RING ABOVE -> "Aa"
+<U00C5> "<U0041><U030A>";"<U0041><U0061>"
+% LATIN SMALL LETTER A WITH RING ABOVE -> "aa"
+<U00E5> "<U0061><U030A>";"<U0061><U0061>"
+% LATIN CAPITAL LETTER O WITH STROKE -> "Oe"
+<U00D8> "<U004F><U0338>";"<U004F><U0065>"
+% LATIN SMALL LETTER O WITH STROKE -> "oe"
+<U00F8> "<U006F><U0338>";"<U006F><U0065>"
+% LATIN CAPITAL LETTER AE -> "Ae"
+<U00C6> "<U0041><U0065>"
+% LATIN SMALL LETTER AE -> "ae"
+<U00E6> "<U0061><U0065>"
+
 translit_end
 END LC_CTYPE
 
